server/operator.go

```diff
@@ -176,16 +182,19 @@ func (co *ChangePeerOperator) Do(region *metapb.Region, leader *metapb.Peer) (bo
 
 // TransferLeaderOperator is used to do leader transfer.
 type TransferLeaderOperator struct {
-	mustSuccess bool
+	count        int
+	maxWaitCount int
 
 	oldLeader *metapb.Peer
 	newLeader *metapb.Peer
 }
 
-func newTransferLeaderOperator(oldLeader, newLeader *metapb.Peer) *TransferLeaderOperator {
+func newTransferLeaderOperator(oldLeader, newLeader *metapb.Peer, waitCount int) *TransferLeaderOperator {
 	return &TransferLeaderOperator{
-		oldLeader: oldLeader,
-		newLeader: newLeader,
+		oldLeader:    oldLeader,
+		newLeader:    newLeader,
+		count:        0,
```

Comment:
`count` and `maxWaitCount` confuse me. 

you can initialize count as waitCount and then use `lto.count < 0 lt.count--` to check.